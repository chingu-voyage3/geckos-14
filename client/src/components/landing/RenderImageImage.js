import React from 'react';
const RenderImageImage = (props) => {
  return (
    <div className="features-three">
      <h2> {props.heading ? props.heading : true} </h2>
      <div className="features-three-container">
        <div>
          <img src={props.imageOne} alt={props.alt} />
        </div>
        <div>
          <img src={props.imageTwo} alt={props.alt} />
        </div>
      </div>
    </div>
  );
};

export default RenderImageImage;
