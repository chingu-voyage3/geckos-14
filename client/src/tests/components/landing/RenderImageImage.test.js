import React from 'react';
import RenderImageImage from '../../../components/landing/RenderTextImage';
import dataOne from '../../../assets/images/dataOne.jpg';
import dataTwo from '../../../assets/images/dataTwo.jpg';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<RenderImageImage
    heading = "Read easy to interpret, live data from all your devices"
    imageOne = {dataOne}
    imageTwo = {dataTwo}
    alt = "Internet of things devices"
   />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
