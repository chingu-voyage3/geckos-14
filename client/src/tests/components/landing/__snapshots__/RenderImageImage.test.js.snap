// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="features-two"
>
  <h2>
     
    Read easy to interpret, live data from all your devices
     
  </h2>
  <div
    className="features-two-container"
  >
    <div>
      <h4>
         
         
      </h4>
    </div>
    <div>
      <img
        alt="Internet of things devices"
      />
    </div>
  </div>
</div>
`;
